nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -126,6 +126,8 @@
 
 - [ToDesk](https://www.todesk.com/linux.html), a remote desktop applicaton. Available as [services.todesk.enable](#opt-services.todesk.enable).
 
+- [meow](https://github.com/PixelSergey/meow), a command-line application to print ASCII cats to your terminal
```

Comment:
This is for NixOS modules, not every packages, otherwise we'll have tons of them

Reply:
Fixed